storybookjs/storybook
# convert state to hooks in color blindness file for a11y addon

Issue:

## What I did
Converted Colorblindness file in a11y addon to use the new react hooks api
## How to test

- Is this testable with Jest or Chromatic screenshots? yes, the tests should be the same
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
